src/contracts/core/DelegationManagerStorage.sol

```diff
@@ -92,6 +96,14 @@ abstract contract DelegationManagerStorage is IDelegationManager {
     /// @notice the address of the StakeRegistry contract to call for stake updates when operator shares are changed
     IStakeRegistryStub public stakeRegistry;
 
+    /// @notice Mapping: AVS => operator => whether or not the operator is currently registered with the AVS
+    mapping(address => mapping(address => bool)) public registeredWithAVS;
```

Comment:
feels maybe a bit odd to define the `OperatorRegistrationStatus` enum but not use it in this mapping / instead use a boolean here.
I think it would be more "harmonious" to use the enum here as well -- clarity seems similar but I think it'd be nice if we end up adding more status in the future?
not feeling strongly / definitely open to other ideas here though